src/app/[...slug]/components/NetworkDashboard/Header/Creation.tsx

```diff
@@ -1,31 +1,25 @@
 import EtherscanLink from "@/components/EtherscanLink";
 import { ProjectCreateEventDocument } from "@/generated/graphql";
-import { useSubgraphQuery } from "@/graphql/useSubgraphQuery";
+import { useBendystrawQuery } from "@/graphql/useBendystrawQuery";
 import { format } from "date-fns";
 import { useJBContractContext } from "juice-sdk-react";
 
 export function Creation() {
   const { projectId } = useJBContractContext();
 
-  const { data: projectCreateEvent } = useSubgraphQuery(
+  const { data: projectCreateEvent } = useBendystrawQuery(
     ProjectCreateEventDocument,
     {
       where: { projectId: Number(projectId) },
     }
   );
 
   const { txHash, timestamp } =
-    projectCreateEvent?.projectEvents?.[0]?.projectCreateEvent ?? {};
+    projectCreateEvent?.projectCreateEvents.items?.[0] ?? {};
```

Comment:
if querying a list, bendystraw responses will include this `items` property which contains the table rows